### DISCUSSION TITLE
Trigger queries through interaction with visualizations

### DISCUSSION THREAD

**Discussion by BeritJanssen** [UUDigitalHumanitieslab/I-analyzer#1418] [Ideas]
Currently, filters affect visualizations, as they trigger a new search. The opposite might also be implemented: visualizations affect filters and trigger a new search. This would mean that:
- [ ] zooming in on the time line affects the date filter
- [ ] clicking on a bar in a histogram selects the associated multiple choice filter option
- [ ] interaction with x / y axis in bar charts sorts data by frequency or key
- [ ] clicking on a line in a neighbouring terms plot triggers `query-term +neighbouring term` search

**Comment by lukavdplas** [comment]
It would be very nice to enable this kind of workflow, but we should take the time to figure out a good control for that. I would find it annoying if zooming always immediately affected the date filter.
